dex/networks/btc/config.go

```diff
@@ -105,10 +114,23 @@ func checkConfig(cfg *Config, name string, network dex.Network, ports NetPorts)
 			}
 			if p != "" {
 				port = p
+				// Patch cfg.RPCPort for consistency with RPCBind.
+				if rpcPort, err := strconv.Atoi(port); err != nil {
+					// port was already parsed as an int by SplitHostPort, but
+					// be cautious.
+					cfg.RPCPort = rpcPort
+				}
```

Comment:
This is the only questionable/unnecessary part of this I think, but it does seem good to generate a consistent struct.